apps/docs/src/components/custom-video-player.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import {Box} from '@primer/react'
+import {ColumnsIcon, PlayIcon} from '@primer/octicons-react'
+
+const CustomVideoPlayer = (props: React.HTMLProps<HTMLVideoElement>) => {
```

Comment:
Non-blocking, but would be amazing if we could use our in-house video player here instead. The branding on it matches Primer (sort of) too. 

A neater idea could even be to just make a VideoPlayer component in our React library, and use it in docs. We'll probably need to provide one for marketing pages eventually anyway.

<img width="883" alt="Screenshot 2023-05-23 at 09 13 34" src="https://github.com/primer/brand/assets/13340707/028a30b7-86a1-4dde-b760-7f41f8ea2da0">

Reply:
For now I just copy-pasted the [custom video player](https://github.com/primer/design/blob/main/src/CustomVideoPlayer.tsx) we use in primer/design for parity. 

I think the best approach would be to add the dotcom video player (or a Brand VideoPlayer if we build it) into `Doctocat` at some point, so we can share it in both documentation sites.